src/cripts/CMakeLists.txt

```diff
@@ -64,6 +66,9 @@ set_target_properties(
 
 install(TARGETS cripts PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/cripts)
 
+# The PUBLIC_HEADER target can't handle include files in a subdirectory, so do those manually.
+install(FILES ${CRIPTS_BUNDLE_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/cripts/Bundles)
```

Comment:
Shouldn't the path be quoted?

Reply:
It wasn't in the install target above (old code), so I assumed you didn't need it. I can certainly add the quotes.

Reply:
There are other paths that might need quoting, so I will make a PR for this separately across all CMakeLists.txt files.